### PR TITLE
Added _y_spt_afterframes_wait

### PR DESCRIPTION
**Pull request by Ramalayha**
Usage: _y_spt_afterframes_wait <delay>

Pauses the processing of afterframes commands for X frames.

### CODE REVIEW

File: spt/OrangeBox/spt-serverplugin.cpp [hunk 1]
```diff
@@ -180,7 +180,7 @@ bool CSourcePauseTool::Load( CreateInterfaceFn interfaceFactory, CreateInterface
 	ConVar_Register(0);
 #endif
 
-	engine = (IVEngineClient*)interfaceFactory(VENGINE_CLIENT_INTERFACE_VERSION, NULL);
+	engine = (IVEngineClient*)interfaceFactory(VENGINE_CLIENT_INTERFACE_VERSION_13, NULL);
```
Comment (YaLTeR) [inline spt/OrangeBox/spt-serverplugin.cpp]: What's this change about?

**Comment by mikesmiffy128** [comment]
Out of curiosity, what exactly would this be used for?

**Comment by YaLTeR** [comment]
Adjusting the script, as in delaying a bunch of stuff at once without having to modify a number on every line.

**Comment by mikesmiffy128** [comment]
Oh yeah, that's a good idea actually. :+1:

**Pull request merged**